sample

```diff
@@ -1 +1,2 @@
 This is Devops sample file
+Adding a new line to sample file
```

Comment:
Thank you so much for adding the line
please add another line also

Reply:
Sure i will do it now 
Thanks Avinash